### PR TITLE
13.0 mig mail activity board

### PR DESCRIPTION
**Pull request by pegonzalezspesol**
Module is now installable after next changes:
- Change version to 13.0.1.0.0
- remove @api.multi
- Do views installable (fix xpath to variables which does not exist anymore)
- Replace sudo(user) by with_user(user)

Question:
- I have not been able to see the button "Activities" in v12. But I do see it in v11. Has somebody an explanation about it?

### CODE REVIEW

**Comment by pegonzalezspesol** [comment]
@mreficent @dajuayen .. somebody reviewing this PR?

File: mail_activity_board/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 David Juaneda - <djuaneda@sdi.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Activities board",
```
Comment (MiquelRForgeFlow) [inline mail_activity_board/__manifest__.py]: maybe we could apply the module name here
```suggestion
    "name": "Mail Activity Board",
```

**Review by MiquelRForgeFlow — APPROVED** [review]
code review :+1:

**Review by emagdalenaC2i — APPROVED** [review]
LGTM

**Comment by pegonzalezspesol** [comment]
thanks @emagdalenaC2i && @mreficent for the fast revision

**Review by angelmoya — APPROVED** [review]

**Pull request merged**